### PR TITLE
suggestion for phase description updates

### PR DESCRIPTION
**Pull request by emmerbodc**
from slocum engineer

Proponents: 
Moderator: @OceanGlidersCommunity/format-mantainers

# Type of PR

- [ ] Typo without possible change of interpretation of the related text.
- [ ] Fix of some error, inconsistency, unforeseen limitation.
- [ ] Style that only affects visually the compiled document
- [X] Addition that does not require change in the current structure.
- [ ] Enhancement that require changes to improve the format.

# Related Issues

# Dates when it got review approvals

# Release checklist

- [ ] Approved by at least two members of the committee?
- [ ] There were modifications after the review approvals? If so, please
      ask reviewers to update their review.
- [ ] Proponents and moderador should explicitly agree that it is ready to
      to merge.
- [ ] The moderador is the one in charge to actually merge or close this PR
      according to the final decision.

# For maintainers

- [ ] Update the moderator with a volunteer from the committee. It would be
      best to have one single moderator to guide and help this PR to move
      forward. It is OK to update the moderador pass it to another one.
- [ ] Confirm that the associated branch was deleted after the merging.
- [ ] Wrap-up and close the related issues.

# Comments

### CODE REVIEW

File: vocabularyCollection/phase.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-| 5 | inflexion | the platform is changing pitch angle to move from a phase to another | pending |
-| 6 | propelled | the platform momentum is increased by the propeller | pending |
-| 7 | transition | the platform is changing from one phase to another | pending |
+| 0 | unknown | when the platform phases describe below do not fit the platform behaviour, or when the behaviour of the platform is unknown | pending |
+| 1 | ascent | the platform is moving up - climbing | pending |
+| 2 | descent | the platform is moving down - diving | pending |
+| 3 | surfacing | the platform is at the surface for communication, recovery or safety purpose | pending |
+| 4 | parking | the platform is maintained at a parking or hover depth | pending |
+| 5 | inflexion | the platform is changing from dive to climb or climb to dive | pending |
+| 6 | propelled | the platform momentum is increased by the propeller(s) or thruster(s) | pending |
```
Comment (callumrollo) [inline vocabularyCollection/phase.md:9]: "momentum is increased" would be incorrect if the glider is e.g. moving at a steady speed. I would use the broader, more accurate description "the platform is underway using propeller(s)/thruster(s)"

Comment (emmerbodc) [reply vocabularyCollection/phase.md]: changed text

**Comment by callumrollo** [comment]
@vturpin @castelao @justinbuck can we get some reviews on this, #259  and #260 please?

**Review by castelao — APPROVED** [review]
This is much better! Thanks @emmerbodc .

**Pull request merged**